requirements.txt

```diff
@@ -3,3 +3,4 @@ logfury>=0.1.2
 requests>=2.9.1
 six>=1.10
 tqdm>=4.5.0
+futures>=3.0.5; python_version < '3.2'
```

Comment:
why is this better?

Reply:
@ppolewicz I don't feel very strongly about this, but I tried to install dependencies using `pip install -r requirements.txt` under Python 2.7 and the `futures` dependency wasn't installed, since it's only specified in setup.py. PEP 508 seems to be the official recommended way to specify version dependent dependencies.